packages/wdio-webdriver-mock-service/src/index.ts

```diff
@@ -11,6 +11,10 @@ const ELEMENT_REFETCHED = '80d860d0-b829-f540-812e-7078eb983795'
 const ELEMENT_ALT = '8bf4d107-a363-40d1-b823-d94bdbc58afb'
 
 export default class WebdriverMockService {
+    mock!: WebDriverMock
```

Comment:
```suggestion
    mock: WebDriverMock
```

Reply:
maybe 
```suggestion
    mock: typeof WebDriverMock
```

Reply:
same issue with this change as well

Reply:
not sure what the solution here should be but `mock` needs to be types as an instance from `WebDriverMock`

Reply:
We can initialize the mock as `mock: WebDriverMock = new WebDriverMock()` or we can add the below propert in tsconfig.json file
`"strictPropertyInitialization": false`

Reply:
ah now I understand, let's move `this.mock = new WebDriverMock()` into the constructor then.